install_config/topics/ocp_osp_admin_tasks.adoc

```diff
@@ -30,6 +30,72 @@ $ openstack user create --password *<password> <username>*
 $ openstack role add --user <username> --project <project> _member_
 ----
 
+The default quotas assigned to new RH OSP projects are not high enough for
+{product-title} installations. Increase the quotas to at least 30 security
+groups, 200 security group rules, and 200 ports. If kuryr SDN is enabled, and
+particularly if namespace isolation is used, increase the quotas to at least
```

Comment:
the Kuryr SDN part can be removed as namespace isolation was not supported in 3.10